README.md

```diff
@@ -1,6 +1,8 @@
 new-pykube
 =========
 
+[![PyPI version](https://badge.fury.io/py/new-pykube.svg)](https://badge.fury.io/py/new-pykube) [![Downloads](https://static.pepy.tech/badge/new-pykube/month)](https://pepy.tech/project/new-pykube) ![Python Test](https://github.com/caas-team/new-pykube/actions/workflows/python-tests.yml/badge.svg)
```

Comment:
I would put each one in their own line. It shouldn't change anything, because Github ignores lines in most cases

Reply:
Sounds good